ui/core.js

```diff
@@ -126,6 +128,67 @@ $.fn.extend( {
 				$( this ).removeAttr( "id" );
 			}
 		} );
+	},
+
+	form: function() {
+		var form;
+
+		if ( this[ 0 ].form && typeof this[ 0 ].form !== "string" ) {
+			return this.pushStack( $( this[ 0 ].form ) );
```

Comment:
I know I said we should support `.pushStack()`, but I was only thinking of the public API. This method doesn't need it.